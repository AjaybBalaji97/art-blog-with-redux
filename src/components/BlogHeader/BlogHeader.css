html{
  font-size:35px;
}
.artifactClass{
  font-size: 0.85rem;
  color: aliceblue;
  padding-left:100px;
}
.CultureClass{
  font-size: 0.5rem;
  font-style: italic;
  color: aliceblue;
  padding-left:100px;
}
.header {
    overflow: hidden;
    background-color: black;
  }
  
  /* Style the header links */
  .header a {
    float: left;
    color: white;
    text-align: center;
    padding-top: 12px;
    padding-right: 12px;
    padding-bottom: 12px;
    padding-left: 12px;
    text-decoration: none;
    font-size: 0.5rem; 
    line-height: 25px;
    border-radius: 4px;
    margin-right: 50px;
    margin-left: 50px;
    margin-bottom:2px;
  }
  
  /* Change the background color on mouse-over */
  .header a:hover {
    background-color: #ddd;
    color: black;
  }
  
  /* Style the active/current link*/
  .header a.active {
    background-color: dodgerblue;
    color: white;
  }
  
  /* Float the link section to the right */
  .header-right {
    float: right;
  }
