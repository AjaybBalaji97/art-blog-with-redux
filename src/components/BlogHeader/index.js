import React, { Component } from 'react';
import './BlogHeader.css';

class BlogHeader extends Component {
  render() {
    return (
      <div class="header">
      <span class="artifactClass">The Artifact</span>
      <br/>
      <span class="CultureClass">Culture & Art blog</span>
      <div class="header-right">
      <a  href="">Blog</a>
      <a href="">About</a>
      <a href="">Contact</a>
      </div>
      </div>
    )
  }
}

export default BlogHeader;
