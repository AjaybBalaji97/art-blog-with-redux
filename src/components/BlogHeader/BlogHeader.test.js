import React from 'react';
import renderer from 'react-test-renderer';
import BlogHeader from '.';
import {shallow} from 'enzyme';


describe('BlogHeader',()=>{
    it('should render header',()=>{
        const tree=renderer.create(<BlogHeader  />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});