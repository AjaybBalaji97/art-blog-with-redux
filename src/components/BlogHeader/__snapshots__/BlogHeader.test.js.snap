// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogHeader should render header 1`] = `
<div
  class="header"
>
  <span
    class="artifactClass"
  >
    The Artifact
  </span>
  <br />
  <span
    class="CultureClass"
  >
    Culture & Art blog
  </span>
  <div
    class="header-right"
  >
    <a
      href=""
    >
      Blog
    </a>
    <a
      href=""
    >
      About
    </a>
    <a
      href=""
    >
      Contact
    </a>
  </div>
</div>
`;
